packages/module/patternfly-docs/content/examples/AdvancedStaticEmail.tsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-console */
+import React from 'react';
+import { FeedbackModal} from '@patternfly/react-user-feedback';
+import { Button } from '@patternfly/react-core';
+import feedbackImage from '@patternfly/react-user-feedback/dist/esm/images/rh_feedback.svg';
+
+export const AdvancedStaticEmailExample: React.FunctionComponent = () => {
+   const  [isOpen, setIsOpen] = React.useState<boolean>(false);
+   const handleButtonClicked = () => {setIsOpen(true)}; 
+   return <>
+    <Button onClick={handleButtonClicked}>Show Modal</Button>
+    <FeedbackModal 
+        email='test.user@patternfly.org'
```

Comment:
Right now if you pass in an email to the modal you can't modify the email.  Is this the API we want or do we think it's better to have a separate prop such as disableEmailModification?  This comes from the original implementation in console where they don't allow for the modification of the email because the user is already logged in and they have it.

Reply:
I would think you would want the consumer to control that so I would vote for adding a prop and having the consumer control that.
If you do decide to disable it internally, I would still have a prop to override that behaviour.

Reply:
After speaking with design, we are not going to allow the ability to disable email modification.  We can add this if it comes in as a request.